### PR TITLE
Input component

### PR DESCRIPTION
**Pull request by i-am-a-codegirl**
Please review my input components

this page contains the input component of CSS

live link: https://css-companion.netlify.app/components/input.html

Following are the features of this page

- nav: contains brand name and links to the home page and components library.
- main: contains the documentation of input components with snippets and examples.
- aside: contains links to all my component libraries.
- footer: contains links to my social media
image
image
image

### CODE REVIEW

File: component/input/input.css [hunk 1]
```diff
@@ -0,0 +1,564 @@
...
+    color: var(--secondary-color);
+}
+
+iframe{
+    padding-top: 1rem;
+}
+
+
+
+
```
Comment (sahiba0915) [inline component/input/input.css:153]: remove unwanted spaces

**Review by sahiba0915 — COMMENTED** [review]
everything looks good to me

File: component/input/input.css [hunk 2]
```diff
@@ -0,0 +1,564 @@
...
+}
+
+.current-pg{
+    color: var(--secondary-color);
+}
+
+
+
+nav ul{
+    margin-left: auto;
```
Comment (bhavyaj12) [inline component/input/input.css]: Code indentation

File: component/input/input.css [hunk 3]
```diff
@@ -0,0 +1,564 @@
...
+ 
+    list-style: none;
+    padding: 0.5rem;
+    font-weight: bold;
+   
+}
+li a{
+    text-decoration: none;
+    color: var(--primary-color);
+
```
Comment (bhavyaj12) [inline component/input/input.css]: Remove extra blanks

File: component/input/input.css [hunk 4]
```diff
@@ -0,0 +1,564 @@
...
+aside #hamburger{
+grid-area: ham;
+display: none;
+
+}
+
+h3{
+    font-size: 1.5rem;
+    /* line-height: 2rem; */
```
Comment (bhavyaj12) [inline component/input/input.css]: Remove comments

**Review by bhavyaj12 — APPROVED** [review]
Looks good, added some minor suggestions
In UI, colour contrast can be improved.
Rest is good

File: component/input/input.css [hunk 5]
```diff
@@ -0,0 +1,564 @@
...
+   
+}
+li a{
+    text-decoration: none;
+    color: var(--primary-color);
+    
+}
+
+a:hover{
+    color: #f472b6;
```
Comment (pnchinmay) [inline component/input/input.css:88]: color variables are preferred

File: component/input/input.css [hunk 6]
```diff
@@ -0,0 +1,564 @@
...
+    border-radius: 0.5rem;
+}
+
+.input-text{
+    color: black;
+    border: 2px solid var(--primary-color);
+}
+
+.input-success{
+    color: lawngreen;
```
Comment (pnchinmay) [inline component/input/input.css:204]: these values don't give same result on different browsers
use hex units as variables

**Review by pnchinmay — APPROVED** [review]
Left a few comments
Everything else looks good :100:

**Pull request merged**